amarachukwu-agbo/hello-books
# #158555465 Add components to handle books and pages not found

#### What does this PR do?

Adds components to handle pages and books not found.

#### Description of Task to be completed?
- Add a component to render when book is not found
- Modify 404 component
- Add jsdoc comments


#### How should this be manually tested?

Clone the repo and install application dependencies. Run command  `npm run start:server` and `npm run start:client`. Open the browser on `127.0.0.1:8080/books/1400000`. You should see a page informing you that the book was not found.

#### Any background context you want to provide?

N/A

#### What are the relevant pivotal tracker stories?

#158555465

#### Screenshots (if appropriate)

N/A

#### Questions:
N/A
